code/controllers/subsystem/timer.dm

```diff
@@ -10,7 +10,7 @@ SUBSYSTEM_DEF(timer)
 
 	flags = SS_TICKER|SS_NO_INIT
 
-	var/list/second_queue = list() //awe, yes, you've had first queue, but what about second queue? Contains: /datum/timedevent
+	var/list/datum/timedevent/second_queue = list() //awe, yes, you've had first queue, but what about second queue?
```

Comment:
Don't do this. Typecasting lists does nothing functionally, but makes them a pain in the ass to grep for.